src/helpers/helpers.dom.js

```diff
@@ -132,7 +139,7 @@ export function getMaximumWidth(domNode) {
 
 	var w = clientWidth - paddingLeft - paddingRight;
 	var cw = getConstraintWidth(domNode);
-	return isNaN(cw) ? w : Math.min(w, cw);
+	return typeof cw === 'undefined' || isNaN(cw) ? w : Math.min(w, cw);
```

Comment:
The error of undefined view is moved even furher

Reply:
I don't think this has changed at all. `isNaN(undefined)` returns `true` and `getConstraintWidth` could already return `undefined`. We're just being more explicit now